bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightHandlerBase.java

```diff
@@ -89,6 +89,8 @@ private DeviceType getDeviceModel(ThingTypeUID typeUID) {
             return DeviceType.ceiling3;
         } else if (typeUID.equals(THING_TYPE_CEILING4)) {
             return DeviceType.ceiling4;
+          } else if (typeUID.equals(THING_TYPE_CEILING11)) {
+              return DeviceType.ceiling11;
```

Comment:
This value does not actually appear in https://github.com/openhab/openhab-addons/blob/4ed24f878d72bf1a67d2270a37acf2f37eb3db24/bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/lib/enums/DeviceType.java#L21